src/shared/cfg.py

```diff
@@ -291,20 +291,17 @@ def LOCAL_FLASK_PORT(self):
         """
         return self("LOCAL_FLASK_PORT", 5000, cast=int)
 
+    @property
+    def DYNALITE_URL(self):
+        return self("DYNALITE_URL", "http://127.0.0.1:8000")
```

Comment:
oh it is, lookie here; maybe a convo about how this is being used.

Reply:
Updated accordingly.